gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -124,6 +125,30 @@ public ProjectService(final RestTemplate restTemplate, final AccountService acco
         return new PageBrowser<>(startPage, page -> listProjects(getProjectsUri(userId, page)));
     }
 
+    /**
+     * Get defined page of paged list of projects that user has access to.
+     *
+     * @param account user whose projects will be returned
+     * @param startPage page to be retrieved
+     * @return {@link PageBrowser} list of found projects for given user or empty list
+     */
+    public PageBrowser<Project> listProjects(final Account account, final PageRequest startPage) {
+        notNull(startPage, "startPage");
+        notNull(account, "account");
+        notEmpty(account.getId(), "account.uri");
+        return new PageBrowser<>(startPage, page -> listProjects(getProjectsUri(account.getId(), page)));
+    }
+
+    /**
+     * Get first page (100 results) of projects that user has access to.
```

Comment:
This is not really precise - you're getting page browser with ability to go through all pages ;)

Reply:
Please what should I write there?

Reply:
Let's keep it simple ;) 

```suggestion
     * Get browser of projects that user has access to.
```

Reply:
Let's keep it simple ;)
```suggestion
     * Get browser of projects that given user has access to.
```